src/main/java/com/appraisal/entities/Employee.java

```diff
@@ -0,0 +1,28 @@
+package com.appraisal.entities;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "employees")
+public class Employee extends BaseEntity {
+    private String firstName;
+
+    private String lastName;
+
+    private Timestamp dateEmployed;
```

Comment:
Use LocalDateTime for your datefields or ZoneDateTime for your date fields instead of Timestamp